core/sr-primitives/src/traits.rs

```diff
@@ -23,7 +23,7 @@ use runtime_io;
 #[cfg(feature = "std")] use serde::{Serialize, de::DeserializeOwned};
 use substrate_primitives;
 use substrate_primitives::Blake2Hasher;
-use codec::{Codec, Encode};
+pub use codec::{Codec, Encode, Decode};
```

Comment:
i'm not convinced we should re-export these. `client` already has a `parity_codec` dependency, perhaps it can import from there?

Reply:
agreed and fixed